prime-router/src/main/kotlin/Receiver.kt

```diff
@@ -258,8 +259,14 @@ open class Receiver(
         if (translation is CustomConfiguration) {
             if (this.topic.isUniversalPipeline) {
                 try {
-                    // This is already scheduled for deletion in https://github.com/CDCgov/prime-reportstream/pull/13313
-                    FhirToHl7Converter(translation.schemaName, BlobAccess.defaultBlobMetadata)
+                    // This is already scheduled for deletion in https://github.com/CDCgov/prime-reportstream/pull/13313/files#r1489489429
+                    FhirToHl7Converter(
+                        translation.schemaName,
+                        BlobAccess.BlobContainerMetadata.build(
+                            "metadata",
```

Comment:
Not good to have this string literal sprinkled throughout the codebase but this is our current pattern. Should have a ticket to fix this.